content/docs/tech-resources/deep-links-self-serve.md

```diff
@@ -1130,9 +1130,7 @@ App verification status:
   Status:  always : 200000000
 ```
 
-If your app has more than one associated domain, the status will be "`ask`" if _any_ of them fail to auto-verify. You can use `adb` to check your setup from Android Studio without having to delete/reinstall. For more information on using the `adb` debugger, see [this article](#adb).
-
-> Note: [CloudFlare](#cloudflare) CDN is problematic for Android auto-verify, owing to its use of 301 redirect for files.
+If your app has more than one associated domain, the status will be "`ask`" if _any_ of them fail to auto-verify. You can use `adb` to check your setup from Android Studio without having to delete/reinstall. For more information on using the `adb` debugger, see [this article](#article-on-investigating-android-deep-link-problems-with-adb-https-medium-com-mobile-app-development-publication-unrealized-deeplink-bug-on-many-apps-6ac78a557702).
```

Comment:
This has the same problem as what I mentioned above. A possible work around would be to just link to the external link. Does that work?

Reply:
Yep. will correct this. I'm OOO until Jan 4, but this should not take long!